### PR TITLE
Add view licence presenter

### PR DESCRIPTION
**Pull request by Cruikshanks**
https://trello.com/c/r9UClZob

As part of working towards having a view invoice endpoint, we need to add a `ViewLicencePresenter`. Eventually, the view invoice endpoint will return a response that includes invoice, licence and transaction details. We'll need a presenter for each level to enable that though.

So, this change adds a `ViewLicencePresenter` which is intended to be used by a `ViewInvoicePresenter`.

### CODE REVIEW

**Comment by Cruikshanks** [comment]
As stated in the last commit, we'll cover this code in the tests we will add you the `ViewInvoicePresenter`. So, I'm happy to ignore SonarCloud for now.

**Comment by Cruikshanks** [comment]
@StuAA78 Sorry! It's very likely I'll need to do some re-jigging as this is not based on main, but on the #229 .

When I do that it'll ditch whatever review you give and I'll only have to pester you again. So, I'll wait till the other is merged and then bring this one back online.

File: app/presenters/view_licence.presenter.js
Comment (StuAA78) [inline app/presenters/view_licence.presenter.js:27]: `presenter.go()` doesn't take arguments so we can remove `transaction` from it.

```suggestion
        return presenter.go()
```

**Review by StuAA78 — CHANGES_REQUESTED** [review]
Looks good! 😁 Just a small change needed I think.

File: app/presenters/view_licence.presenter.js
Comment (Cruikshanks) [reply app/presenters/view_licence.presenter.js]: You spotted it! I only did so once I merged these changes into my invoice presenter branch 🤦 😁

**Review by StuAA78 — APPROVED** [review]

**Pull request merged**
